documentation/smg/10_how_to_use_the_SAN_area.md

```diff
@@ -0,0 +1,35 @@
+# How to use the SAN area
+
+ISIS has provided a storage area in which PACE can keep large test data. The
+area should contain data files from which we derive smaller sets of data
+(alongside scripts recording how the smaller sets were generated) and files
+that are too large to store in git repositories.
+
+A FedID is required in order to mount the SAN area. In order to mount the SAN
+on PACE's systems, a dedicated user ID with which to mount the SAN area has
+been provided to PACE.
+
+## Mounting the SAN on Jenkins
+
+To mount the SAN when running a Jenkins pipeline, you should use the dedicated
+user ID provided to PACE. The credentials for this are stored in a credentials
```

Comment:
I would be explicit here on the format of that file so anyone looking at it knows exactly what they're seeing 
```
username\n
domain\n
password\n
```
If we were using the username/password facility of the Jenkins credentials plugin that would give us `MY_ID` (`username:password`), `MY_ID_USR` (`username`) and `MY_ID_PSW` (`password`) so everything is explicit.

Reply:
`stored in a credentials file` possible needs a little more explanation: 
- this is a credentials file stored in the secure Jenkins Credentials Binding plugin
- https://www.jenkins.io/doc/book/using/using-credentials/